charts/paperless/values.yaml

```diff
@@ -0,0 +1,75 @@
+# Default values for Paperless.
+
+image:
+  repository: thepaperlessproject/paperless
+  pullPolicy: IfNotPresent
+  tag: latest
+  args: "['gunicorn', '-b', '0.0.0.0:8000']"
```

Comment:
`args` should really not be a part of `image` but instead it's own parent.

Reply:
Cheers, done